src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -783,18 +783,36 @@ public function prepare_item_for_response( $item, $request ) {
 					}
 
 					$size_data['source_url'] = $image_src[0];
+
+					if ( empty( $size_data['sources'] ) || ! is_array( $size_data['sources'] ) ) {
+						continue;
+					}
+
+					$directory = dirname( $size_data['source_url'] );
+					foreach ( $size_data['sources'] as $mime => &$mime_details ) {
+						$source_url                 = "{$directory}/{$mime_details['file']}";
+						$mime_details['source_url'] = $source_url;
```

Comment:
Nit-pick, but this variable can be removed:
```suggestion
						$mime_details['source_url'] = "{$directory}/{$mime_details['file']}";
```

Reply:
Thanks. This feedback is addressed.